charts/icm/templates/test-job.yaml

```diff
@@ -84,9 +84,12 @@ spec:
         - -c
         - |
         {{- if index .Values "icm-as" "replication" "enabled" }}
-          until wget -qO- http://{{ .Release.Name }}-icm-as-live:7744/status/ReadinessProbe | grep -o "RUNNING" && wget -qO- http://{{ .Release.Name }}-icm-as-edit:7744/status/ReadinessProbe | grep -o "RUNNING"; do
+          until wget --server-response --spider http://{{ .Release.Name }}-ingress-nginx-controller/healthz 2>&1 | grep "HTTP/" | awk '{print $2}' | grep -q "200" &&
```

Comment:
If only the HTTP response code of the ingress nginx controller health endpoint is of interest, as a suggestion it's also possible to get that directly with `curl`:
`until curl --silent --output /dev/null --write-out "%{response_code}" http://{{ .Release.Name }}-ingress-nginx-controller/healthz | grep -q "200"`